documentation/assemblies/deploying/assembly-kraft-mode.adoc

```diff
@@ -0,0 +1,62 @@
+// This assembly is included in the following assemblies:
+//
+// deploying/assembly-deploy-kafka-cluster.adoc
+
+[id='assembly-kraft-mode-{context}']
+= Using Kafka in KRaft mode
+
+[role="_abstract"]
+KRaft (Kafka Raft metadata) mode replaces Kafka's dependency on ZooKeeper for cluster management. 
+KRaft mode simplifies the deployment and management of Kafka clusters by bringing metadata management and coordination of clusters into Kafka.
+
+Kafka in KRaft mode is designed to offer enhanced reliability, scalability, and throughput.
+Metadata operations become more efficient as they are directly integrated.
+And by removing the need to maintain a ZooKeeper cluster, there's also a reduction in the operational and security overhead.
+
+To deploy a Kafka cluster in KRaft mode, you must use `Kafka` and `KafkaNodePool` custom resources.
+The `Kafka` resource using KRaft mode must also have the annotation `strimzi.io/kraft: enabled`.
+For more details and examples, see xref:deploying-kafka-cluster-kraft-{context}[].
+
+Through xref:config-node-pools-{context}[node pool configuration using `KafkaNodePool` resources], nodes are assigned the role of broker, controller, or both:
+
+* *Controller* nodes operate in the control plane to manage cluster metadata and the state of the cluster using a Raft-based consensus protocol.
+* *Broker* nodes operate in the data plane to manage the streaming of messages, receiving and storing data in topic partitions.
+* *Dual-role* nodes fulfill the responsibilities of controllers and brokers.
+
+Controllers use a metadata log, stored as a single-partition topic (`__cluster_metadata`) on every node, which records the state of the cluster. 
+An active lead controller manages updates to the metadata log, and follower controllers replicate these updates. 
+The metadata log stores information on brokers, replicas, topics, and partitions, including the state of in-sync replicas and partition leadership. 
+Kafka uses this metadata to coordinate changes and manage the cluster effectively.
+
+Brokers store the metadata log passively to stay up-to-date with the cluster's state.
```

Comment:
I would also add some terminology used for broker ... "they act as observers" (as you say before that controllers are followers of the leader).